mate-panel/panel-run-dialog.c

```diff
@@ -932,14 +968,17 @@ panel_run_dialog_add_items_idle (PanelRunDialog *dialog)
 		gicon = g_app_info_get_icon(G_APP_INFO(ginfo));
 
 		gtk_list_store_append (dialog->program_list_store, &iter);
-		gtk_list_store_set (dialog->program_list_store, &iter,
-				    COLUMN_GICON,     gicon,
-				    COLUMN_NAME,      g_app_info_get_display_name(G_APP_INFO(ginfo)),
-				    COLUMN_COMMENT,   g_app_info_get_description(G_APP_INFO(ginfo)),
-				    COLUMN_EXEC,      g_app_info_get_commandline(G_APP_INFO(ginfo)),
-				    COLUMN_PATH,      matemenu_tree_entry_get_desktop_file_path (entry),
-				    COLUMN_VISIBLE,   TRUE,
-				    -1);
+                gtk_list_store_set (dialog->program_list_store, &iter,
+                                   COLUMN_GICON,     gicon,
+                                   COLUMN_NAME,      g_app_info_get_display_name(G_APP_INFO(ginfo)),
+                                   COLUMN_ACCELERATOR_MASK, (gint)accelerator_key_mapping[i].modifier,
+                                   COLUMN_ACCELERATOR_KEY_VALUE, accelerator_key_mapping[i].key_id,
+                                   COLUMN_COMMENT,   g_app_info_get_description(G_APP_INFO(ginfo)),
+                                   COLUMN_EXEC,      g_app_info_get_commandline(G_APP_INFO(ginfo)),
+                                   COLUMN_PATH,      matemenu_tree_entry_get_desktop_file_path (entry),
+                                   COLUMN_VISIBLE,   TRUE,
+                                   -1);
```

Comment:
Indentation has a problem here, it doesn't align properly it seems

Reply:
I'm sorry my editor was not properly setup. I created a clang-format file to fix these issues. If you want, I can share it in some form.